x/incentives/keeper/gauge.go

```diff
@@ -463,3 +461,26 @@ func (k Keeper) chargeFeeIfSufficientFeeDenomBalance(ctx sdk.Context, address sd
 	}
 	return nil
 }
+
+// initGaugeInfo takes in a list of pool IDs and a splitting policy and returns a InternalGaugeInfo struct with weights initialized to zero.
+// Returns error if fails to retrieve gauge ID for a pool.
+func (k Keeper) initGaugeInfo(ctx sdk.Context, poolIds []uint64) (types.InternalGaugeInfo, error) {
+	gaugeRecords := make([]types.InternalGaugeRecord, 0, len(poolIds))
+	for _, poolID := range poolIds {
+		gaugeID, err := k.pik.GetInternalGaugeIDForPool(ctx, poolID)
+		if err != nil {
+			return types.InternalGaugeInfo{}, err
+		}
+
+		gaugeRecords = append(gaugeRecords, types.InternalGaugeRecord{
+			GaugeId:          gaugeID,
+			CurrentWeight:    osmomath.ZeroInt(),
+			CumulativeWeight: osmomath.ZeroInt(),
+		})
+	}
+
+	return types.InternalGaugeInfo{
+		TotalWeight:  osmomath.ZeroInt(),
+		GaugeRecords: gaugeRecords,
+	}, nil
+}
```

Comment:
I might be reading the comment wrong, but I don't see a splitting policy as a required input.

Reply:
I ended up removing it but missed the comment update. Thanks for catching it.

I will make this change in a subsequent PR to avoid unnecessary merge conflicts in a PR sequence

Reply:
Tracked here just in case: https://github.com/osmosis-labs/osmosis/issues/6057

Reply:
addressed: https://github.com/osmosis-labs/osmosis/pull/6525